### PR TITLE
#611 Booksley case study - desktop

### PR DESCRIPTION
**Pull request by szczepanbarszczowski**

### CODE REVIEW

File: src/routes/Booksley/Booksley.js [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+      </div>
+
+      <hr className={c.hr} />
+
+      <div className={c.eightSection}>
+        <div className={c.textWrapper}>
+          <div className={cn(c.title, c.booksTitle)}>
+            Swap books with<br />other book-lovers
+          </div>
+          <div className={cn(c.subtitle)}>The community</div>
```
Comment (CodeMeNatalie) [inline src/routes/Booksley/Booksley.js]: no need of `cn`

File: src/routes/Booksley/Booksley.js [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+import Arrows from './assets/arrows.svg'
+import Book from './assets/book.svg'
+import Bi from './assets/bi.svg'
+
+import addBookPreview from './assets/addBookPreview.png'
+import secondPreview from './assets/secondCover.png'
+
+import addBookVideo from './assets/addbook-trimmed.mp4'
+import bookSwapVideo from './assets/booksley-bookswap.mp4'
+
```
Comment (CodeMeNatalie) [inline src/routes/Booksley/Booksley.js]: 😻

File: src/routes/Booksley/Booksley.module.scss [hunk 3]
```diff
@@ -0,0 +1,370 @@
+@import '~styles/base';
+
+.firstSection {
+  @include container();
```
Comment (CodeMeNatalie) [inline src/routes/Booksley/Booksley.module.scss]: `@include container;`

File: src/routes/Booksley/Booksley.module.scss [hunk 4]
```diff
@@ -0,0 +1,370 @@
...
+
+  .heading {
+    padding-top: 100px;
+  }
+
+  .deliverables {
+    @include flex(row);
```
Comment (CodeMeNatalie) [inline src/routes/Booksley/Booksley.module.scss]: `@include flex;`

File: src/routes/Booksley/Booksley.module.scss [hunk 5]
```diff
@@ -0,0 +1,370 @@
...
+    padding-top: 69px;
+  }
+}
+
+.secondSection {
+  @include container();
```
Comment (CodeMeNatalie) [inline src/routes/Booksley/Booksley.module.scss]: `@include container;`

File: src/routes/Booksley/Booksley.module.scss [hunk 6]
```diff
@@ -0,0 +1,370 @@
...
+    .booksSubtitle {
+      padding-bottom: 43px;
+    }
+  }
+}
+
+.thirdSectionWrapper {
+  background-color: $color-black;
+  height: 961px;
+  @include flex(row, center, center);
```
Comment (CodeMeNatalie) [inline src/routes/Booksley/Booksley.module.scss]: mixins before declarations

File: src/routes/Booksley/Booksley.module.scss [hunk 7]
```diff
@@ -0,0 +1,370 @@
...
+    padding-top: 99px;
+    padding-bottom: 92px;
+  }
+}
+
+.eleventhSection {
+  @include relative;
+
+  width: 100%;
+  height: 900px;
```
Comment (CodeMeNatalie) [inline src/routes/Booksley/Booksley.module.scss]: `@include size(100%, 900px);`

File: src/routes/Booksley/Booksley.module.scss [hunk 8]
```diff
@@ -0,0 +1,370 @@
...
+  max-height: 900px;
+  background-color: $color-image-background;
+
+  .wrapper {
+    @include container;
+    z-index: 2;
```
Comment (CodeMeNatalie) [inline src/routes/Booksley/Booksley.module.scss]: can you use mixin for z-index here?

File: src/routes/Booksley/Booksley.module.scss [hunk 9]
```diff
@@ -0,0 +1,370 @@
...
+
+    .kellyWrapper {
+      @include flex(column, flex-start);
+
+      padding-left: 250px;
+      padding-top: 243px;
+      z-index: 1;
```
Comment (CodeMeNatalie) [inline src/routes/Booksley/Booksley.module.scss]: can you use mixin for z-index here?

File: src/routes/Booksley/Booksley.module.scss [hunk 10]
```diff
@@ -0,0 +1,370 @@
...
+  .titleWrap {
+    padding-top: 181px;
+    padding-bottom: 116px;
+    width: 100%;
+  }
+
+  .projectsWrap {
+    @include flex(row);
+
+    padding-bottom: 200px;
```
Comment (CodeMeNatalie) [inline src/routes/Booksley/Booksley.module.scss]: `@include pb(7);`

File: src/routes/Booksley/Booksley.module.scss [hunk 11]
```diff
@@ -0,0 +1,370 @@
...
+  }
+}
+
+.element {
+  &:first-child {
+    padding-right: 30px;
```
Comment (CodeMeNatalie) [inline src/routes/Booksley/Booksley.module.scss]: `@include pr(2);`

File: src/routes/Booksley/Booksley.module.scss [hunk 12]
```diff
@@ -0,0 +1,370 @@
...
+  }
+
+  .image {
+    width: 630px;
```
Comment (CodeMeNatalie) [inline src/routes/Booksley/Booksley.module.scss]: `@include size(630px);`

File: src/routes/Booksley/Booksley.module.scss [hunk 13]
```diff
@@ -0,0 +1,370 @@
...
+  line-height: 30px;
+}
+
+.textWhite {
+  color: $color-white;
+}
+
+.imageWrapper {
+  width: 100%;
+  height: auto;
```
Comment (CodeMeNatalie) [inline src/routes/Booksley/Booksley.module.scss]: `@include size(100%, auto);`

File: src/routes/Booksley/Booksley.module.scss [hunk 14]
```diff
@@ -0,0 +1,370 @@
...
+  z-index: 1;
```
Comment (CodeMeNatalie) [inline src/routes/Booksley/Booksley.module.scss]: z index mixin

File: src/routes/Booksley/Booksley.module.scss [hunk 15]
```diff
@@ -0,0 +1,370 @@
...
+  }
+
+  .itemSubName {
+    @include font(h6, $color-gray-dark, $font-family: $font-family-light);
+    letter-spacing: 0;
+    font-weight: 300;
+  }
+
+  .arrowButton {
+    padding: 0;
```
Comment (CodeMeNatalie) [inline src/routes/Booksley/Booksley.module.scss]: `@include p(0);`

File: src/routes/Booksley/Booksley.module.scss [hunk 16]
```diff
@@ -0,0 +1,370 @@
...
+}
+
+.container {
+  @include container;
+}
+
+.col {
+  @include col(6, $gutter: false);
+}
+
```
Comment (CodeMeNatalie) [inline src/routes/Booksley/Booksley.module.scss]:
```
.smallTitle, .smallSubtitle {
font-size: 16px;
line-height: 24px;
letter-spacing: 0;
}

.smallTitle {
font-family: $font-family-medium;
padding-top: 40px;
}

.smallSubtitle {
font-family: $font-family-light;
}

```

File: src/routes/Booksley/Booksley.module.scss [hunk 17]
```diff
@@ -0,0 +1,370 @@
...
+.videoWrapper {
+  @include relative;
+}
+
+.video {
+  @include absolute(top 86px bottom 0 left 64px right 0);
+
+  width: 320px;
+  height: 568px;
```
Comment (CodeMeNatalie) [inline src/routes/Booksley/Booksley.module.scss]: mixin size and index

File: src/routes/Booksley/Booksley.module.scss [hunk 18]
```diff
@@ -0,0 +1,370 @@
...
+.avatarWrapper {
+  @include flex(row, center, center);
+
+  .titlesWrapper {
+    @include flex(column, flex-start, center);
+
+    padding-left: 23px;
+
+    .smallTitle {
+      padding-top: 0;
```
Comment (CodeMeNatalie) [inline src/routes/Booksley/Booksley.module.scss]: `@include p(0);`

File: src/routes/Booksley/assets/arrows.svg [hunk 19]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="91px" height="88px" viewBox="0 0 91 88" version="1.1" xmlns="http://www.w3.org/2000/svg"
+     xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: sketchtool 50.2 (55047) - http://www.bohemiancoding.com/sketch -->
```
Comment (CodeMeNatalie) [inline src/routes/Booksley/assets/arrows.svg]: remove unnecessary info (ie. comments, title, empty defs, etc)

File: src/routes/Booksley/assets/bi.svg [hunk 20]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="59px" height="72px" viewBox="0 0 59 72" version="1.1" xmlns="http://www.w3.org/2000/svg"
+     xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: sketchtool 50.2 (55047) - http://www.bohemiancoding.com/sketch -->
```
Comment (CodeMeNatalie) [inline src/routes/Booksley/assets/bi.svg]: remove unnecessary info (ie. comments, title, empty defs, etc)

File: src/routes/Booksley/assets/book.svg [hunk 21]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="94px" height="67px" viewBox="0 0 94 67" version="1.1" xmlns="http://www.w3.org/2000/svg"
+     xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: sketchtool 50.2 (55047) - http://www.bohemiancoding.com/sketch -->
```
Comment (CodeMeNatalie) [inline src/routes/Booksley/assets/book.svg]: remove unnecessary info (ie. comments, title, empty defs, etc)

File: src/routes/Booksley/Booksley.js (hunk 1 above)
Comment (szczepanbarszczowski) [reply src/routes/Booksley/Booksley.js]: +1

File: src/routes/Booksley/Booksley.js (hunk 2 above)
Comment (szczepanbarszczowski) [reply src/routes/Booksley/Booksley.js]: +1

File: src/routes/Booksley/Booksley.module.scss (hunk 3 above)
Comment (szczepanbarszczowski) [reply src/routes/Booksley/Booksley.module.scss]: +1

File: src/routes/Booksley/Booksley.module.scss (hunk 5 above)
Comment (szczepanbarszczowski) [reply src/routes/Booksley/Booksley.module.scss]: +1

File: src/routes/Booksley/Booksley.module.scss (hunk 17 above)
Comment (szczepanbarszczowski) [reply src/routes/Booksley/Booksley.module.scss]: +1

File: src/routes/Booksley/Booksley.module.scss (hunk 18 above)
Comment (szczepanbarszczowski) [reply src/routes/Booksley/Booksley.module.scss]: +1

File: src/routes/Booksley/assets/arrows.svg (hunk 19 above)
Comment (szczepanbarszczowski) [reply src/routes/Booksley/assets/arrows.svg]: +1

File: src/routes/Booksley/assets/bi.svg (hunk 20 above)
Comment (szczepanbarszczowski) [reply src/routes/Booksley/assets/bi.svg]: +1

File: src/routes/Booksley/assets/book.svg (hunk 21 above)
Comment (szczepanbarszczowski) [reply src/routes/Booksley/assets/book.svg]: +1

File: src/routes/Booksley/Booksley.module.scss (hunk 6 above)
Comment (szczepanbarszczowski) [reply src/routes/Booksley/Booksley.module.scss]: +1

File: src/routes/Booksley/Booksley.module.scss (hunk 7 above)
Comment (szczepanbarszczowski) [reply src/routes/Booksley/Booksley.module.scss]: +1

File: src/routes/Booksley/Booksley.module.scss (hunk 8 above)
Comment (szczepanbarszczowski) [reply src/routes/Booksley/Booksley.module.scss]: +1

File: src/routes/Booksley/Booksley.module.scss (hunk 9 above)
Comment (szczepanbarszczowski) [reply src/routes/Booksley/Booksley.module.scss]: +1

File: src/routes/Booksley/Booksley.module.scss (hunk 10 above)
Comment (szczepanbarszczowski) [reply src/routes/Booksley/Booksley.module.scss]: +1

File: src/routes/Booksley/Booksley.module.scss (hunk 12 above)
Comment (szczepanbarszczowski) [reply src/routes/Booksley/Booksley.module.scss]: +1

File: src/routes/Booksley/Booksley.module.scss (hunk 13 above)
Comment (szczepanbarszczowski) [reply src/routes/Booksley/Booksley.module.scss]: +1

File: src/routes/Booksley/Booksley.module.scss (hunk 14 above)
Comment (szczepanbarszczowski) [reply src/routes/Booksley/Booksley.module.scss]: +1

File: src/routes/Booksley/Booksley.module.scss (hunk 15 above)
Comment (szczepanbarszczowski) [reply src/routes/Booksley/Booksley.module.scss]: +1

File: src/routes/Booksley/Booksley.module.scss (hunk 16 above)
Comment (szczepanbarszczowski) [reply src/routes/Booksley/Booksley.module.scss]: +1

**Pull request merged**